packages/compass-indexes/src/stores/store.js

```diff
@@ -48,20 +48,20 @@ const configureStore = (options = {}) => {
     const globalAppRegistry = options.globalAppRegistry;
     store.dispatch(globalAppRegistryActivated(globalAppRegistry));
 
-    const deploymentAwarenessStore = globalAppRegistry.getStore(
-      'DeploymentAwareness.WriteStateStore'
-    );
+    const instanceStore = globalAppRegistry.getStore('App.InstanceStore');
+    const instance = instanceStore.getState().instance;
 
-    deploymentAwarenessStore.listen((state) => {
-      store.dispatch(writeStateChanged(state.isWritable));
-      store.dispatch(getDescription(state.description));
-    });
+    // set the initial values
+    store.dispatch(writeStateChanged(instance.isWritable));
+    store.dispatch(getDescription(instance.description));
 
-    // Load the initial deployment awareness state.
-    store.dispatch(
-      writeStateChanged(deploymentAwarenessStore.state.isWritable)
-    );
-    store.dispatch(getDescription(deploymentAwarenessStore.state.description));
+    // these can change later
+    instance.on('change:isWritable', () => {
+      store.dispatch(writeStateChanged(instance.isWritable));
+    });
+    instance.on('change:description', () => {
+      store.dispatch(getDescription(instance.description));
+    });
```

Comment:
I kinda went back and forth on this. It shouldn't be possible for these scoped plugins for the version to change afterwards because they only get instantiated after we received the fully loaded instance with details. So I guess the answer is to remove it elsewhere.

Reply:
Actually no I think they get an instance but the details can arrive later 🤔 Now I'm confused again.

Reply:
(which I've done now)